text_prediction.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+"""text_prediction.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1lxX2CA_qdB8PDoFZ-eZ-bnm0j9TXtJuV
+"""
+
+# https://www.tensorflow.org/tutorials/text/text_generation
+
+import tensorflow as tf
+
+import numpy as np
+import os
+import time
+
+path_to_file = tf.keras.utils.get_file('shakespeare.txt', 'https://storage.googleapis.com/download.tensorflow.org/data/shakespeare.txt')
```

Comment:
@ShakeelAlibhai in this `get_file` method. would you be able to add more than one file. Or is it one file path to train and thats it?